homeassistant/components/rachio/services.yaml

```
@@ -1,33 +1,68 @@
 set_zone_moisture_percent:
-  description: Set the moisture percentage of a zone or group of zones.
+  name: Set Zone Moisture Percent
+  description: Set the moisture percentage of a zone or list of zones.
+  target:
+    entity:
+      integration: rachio
+      domain: switch
   fields:
-    entity_id:
-      description: Name of the zone entity. Can also be a group of zones. [Required]
-      example: "switch.front_yard"
     percent:
-      description: Set the desired zone moisture percentage from 0 to 100. [Required]
+      name: Percent
+      description: Set the desired zone moisture percentage from 0 to 100.
+      required: true
       example: 50
+      default: 50
+      selector:
+        number:
+          min: 0
+          max: 100
+          mode: slider
+          unit_of_measurement: "%"
+          step: 1
 start_multiple_zone_schedule:
-  description: Create a custom schedule of zones and runtimes.
+  name: Start Multiple Zones
+  description: Create a custom schedule of zones and runtimes. Note that all zones should be on the same controller to avoid issues.
+  target:
+    entity:
+      integration: rachio
+      domain: switch
   fields:
-    entity_id:
-      description: Name of the zone or zones to run. Zones should all be on the same controller, attempting to start zones on multiple controllers may have undesired results. [Required]
-      example: "switch.front_yard, switch.side_yard"
     duration:
-      description: Number of minutes to run the zone(s). If only 1 duration is given, that time will be used for all zones. If given a list of durations, the durations will apply to the respective zone listed above. [Required]
+      name: Duration
+      description: Number of minutes to run the zone(s). If only 1 duration is given, that time will be used for all zones. If given a list of durations, the durations will apply to the respective zones listed above.
       example: 15, 20
+      required: true
+      selector:
+        text:
```

This can also be a list, thus it should be an `object`.

```suggestion
        object:
```

---

👍 Thanks!